<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawWindow</class>
 <widget class="QDialog" name="WithdrawWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: #f5f7fa;
}
QMessageBox {
    color: #2c3e50; 
    background-color: #ecf0f1; 
}

QMessageBox QLabel {
    color: #2c3e50; 
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #2c3e50;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Withdraw Window</string>
   </property>
  </widget>
  <widget class="QLabel" name="AccNumLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #2c3e50;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Account Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="BalanceLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #2c3e50;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Balance:   </string>
   </property>
  </widget>
  <widget class="QLabel" name="UsernamLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #2c3e50;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Username:  </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Amount_inp">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ecf0f1;
    border: 1px solid #bdc3c7;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
	color: #2c3e50;	
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmWith">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 10px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1c5985;
}</string>
   </property>
   <property name="text">
    <string>Confirm Withdraw</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #2c3e50;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 10px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1c5985;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
